src/libs/tagging.c

```diff
@@ -751,6 +873,21 @@ static void rename_button_clicked(GtkButton *button, dt_lib_module_t *self)
   if (gtk_dialog_run(GTK_DIALOG(dialog)) == GTK_RESPONSE_YES)
   {
     const char *newtag = gtk_entry_get_text(GTK_ENTRY(entry));
+    char *message = NULL;
+    if (!newtag[0])
+      message = _("null string is not allowed. Aborting");
+    if(strchr(newtag, '|') != 0)
+      message = _("'|' character is not allowed for renaming tag.\nTo modify the hierachy use rename path instead. Aborting.");
```

Comment:
To -> to (no capital letter). Likewise for Aborting -> aborting.

Reply:
done